phil/s013-course-periods.md

```diff
@@ -13,7 +44,7 @@ Screens:
 ```coffee
```

Comment:
Right now we're simply using the lowercase first word of the course title for styling, which is admittedly pretty dumb.

I'd suggest we add a `subject_matter` attribute to courses.  If we think there could be additional related properties, it could be tucked inside a `metadata` property.